build.sbt

```diff
@@ -53,6 +53,7 @@ lazy val runner = Project(
   ).settings(standardSettings ++ bintrarySettings ++ Seq(
     name := "hedgehog-runner"
   ) ++ Seq(libraryDependencies ++= Seq(
+      "com.github.scopt" %% "scopt" % "4.0.0-RC2"
```

Comment:
Thank you (again) for all the code contributions. I really appreciate it, and the time you've taken to do the work.

I'm really sorry to do this, but I'd very much like to keep hedgehog dependency free given (we've made it this far without scalaz/cats). Even if it means a bit of clunky/hacky pattern matching, I think it's worth that pain.

I'd be happy to help with that if you want a hand, I feel bad for making you do that extra work to make scala-hedgehog much nicer and more usable. :(

Reply:
> Just to double check before I invest the time, you would be ok with this change if I implement the option parsing without using a third-party library?

I would indeed!

For some background what I eventually want would be to add tag support and to be able to filter out tests that way (and to be able to reuse that arg parsing from SBT too).

Let me know if you need a hand. If I had the time I would probably write a micro opt-parse like module, but for 3 args at the moment you can probably do a simple pattern match and recurse. I'm sorry again to make you jump through that hoop.